packages/svelte/src/internal/client/dom/elements/transitions.js

```diff
@@ -439,7 +443,12 @@ function animate(element, options, counterpart, t2, on_finish, on_abort) {
 
 	return {
 		abort: () => {
-			animation?.cancel();
+			if (animation) {
+				animation.cancel();
+				// Force cleanup on properties that can retain memory
+				animation.effect = null;
+				animation.timeline = null;
```

Comment:
I'm surprised this is necessary — what if we do `animation = null` instead so the entire thing can be GC'd?

Reply:
`animation = null` doesn't do anything, I tried